next-docs/public/examples/tabsTW/SegmentControl.tsx

```diff
@@ -0,0 +1,60 @@
+import { Tab } from '@headlessui/react';
+import React, { Fragment, useState } from 'react';
+
+const items = ['$Link', '$Link', '$Link', '$Link'];
+
+const Example = () => {
+  return (
+    <>
+      <Tab.Group>
+        <Tab.List className="flex h-10 p-1 bg-beerus rounded-xl">
+          {items.map((item: string) => {
+            return (
+              <div className="mr-1">
```

Comment:
Also, it's always better to use ms, ps, me, and pe. Instead of corresponding ml, pl, mr, and pr. Start and end are more suitable for rtl